docs/whitepaper.tex

```diff
@@ -220,9 +247,118 @@ \subsection{DEX pool}
+			\item The input and output pools are both live
+		\end{itemize}
+	\item When the redeemer action is Liquidity
+		\begin{itemize}
+			\item The liquidity invariant $L^2/k^2$ is non decreasing before fees
+			and appropriate fees are paid.
+				\todo{how should fees work here?}
+			\item The datum correctly reflects the change in issued liquidity
+			\item The ``liveness" of the pool is not changed
+			\item If the input pool is dead the issued liquidity decreases
+			\item If the input pool is dead the withdraw is balanced
+		\end{itemize}
+	\item When the redeemer action is Kill
+		\begin{itemize}
+			\item The admin signed the transaction
+			\item The output pool is dead
+			\item Nothing else is changed in the pool
+		\end{itemize}
+	\item When The redeemer action is Collect
+		\begin{itemize}
+			\item The admin fees in the output pool are zero
+			\item the non admin fees balance stays the same
+			\item the output value remains correct for the datum
+			\item The admin keys signed the tx
+		\end{itemize}
+\end{itemize}
+
+
+\subsection{Pool id minting policy}
+
+The pool id token is a token used to track pools
+and to ensure that the pools the protocol interacts with were opened legitimately.
+The pool id token's minting policy is responsible for ensuring
+that it is only minted as part of opening a valid vault.
+To do so it should enforce all of the following:
+
+\begin{itemize}
+	\item The config utxo is valid
+	\item Exactly one token is minted
+	\item That token is sent to a pool
+	\item The redeemer provides a seed utxo
+	\item The token name of the minted token is the hash of the seed utxo
+	\item The seed utxo is spent in the transaction
+	\item The output pool datum is valid
+	\item The datum correctly reflects the number of liquidity tokens being minted
+	\item at least one liquidity token is being minted
+	\item the initial invariant is not zero ie. some liquidity is provided in each token
+	\item The issued liquidity is exactly the square root of the initial value
+\end{itemize}
+
+NOTE: Pools not being duplicates will be checked in the browser app, but not enforced onchain.
+
+\subsection{Liquidity token minting policy}
+
+The liquidity tokens are used to represent ownership of liquidity in a pool.
+Their minting policy is responsible for ensuring that they are only minted
+during liquidity interactions with a pool and during pool initializations.
+
+Since the Pools Address Validator and the pool id token minting policy
+already enforce the logic regarding how much should be minted the liquidity
+token minting policy doesn't have to do much. It's main responsibility is enforcing that
+some pool is involved either by being opened or an existing pool being used, and that
+the liquidity tokens being created correspond to the pool being used.
+The way we keep track of which liquidity tokens correspond to which pools
+is by having the token name be the id of the pool the liquidity tokens
+belong to.
+
```

Comment:
Nitpick: New paragraphs should leave a newline, using `\\`, to make our formatting consistent.